src/wazuh_modules/vulnerability_scanner/tests/policyManager/policyManager_test.cpp

```diff
@@ -0,0 +1,196 @@
+/*
+ * Wazuh PolicyManager
+ * Copyright (C) 2015, Wazuh Inc.
+ * September 8, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+#include "policyManager_test.h"
+
+void PolicyManagerTest::SetUp()
+{
+    m_policyManager = std::make_unique<PolicyManager>();
+};
+
+void PolicyManagerTest::TearDown() {};
+
+TEST_F(PolicyManagerTest, validConfigurationVulnerabilityDetection)
+{
+    const auto& configJson {nlohmann::json::parse(R"({
+    "enabled": "yes",
+    "index-status": "yes",
+    "feed-update-interval": "60m",
+    "offline-url": "file:///var/algo.tar.gz"
+})")};
+    EXPECT_NO_THROW(m_policyManager->validateAndConfigureVulnerabilityDetection(configJson));
```

Comment:
We should validate the values parsed for each json key using the getters functions.

Reply:
I don't think is necessary if the test objective is an invalid configuration, because first, we check if the values that are mandatory are present in the configuration and throw an exception if not.

Reply:
With this check, you know that something is there and that something is according to the field expectations, but you don't assure that each field has the value that is expected to have. That's the point of my comment.

Reply:
Perfect, added =)